vsintegration/src/vs/FsPkgs/FSharp.Project/Common.Source.CSharp/Project/HierarchyNode.cs

```diff
@@ -3478,12 +3459,6 @@ private int CreateResourceDocDataHelper(FileNode f, uint itemidResource, out IVs
             }
             finally
```

Comment:
If we are removing the comment from the finally and no code is left.  Perhaps we should cleanup the try/finally too.